okhttp/src/main/java/io/grpc/okhttp/SslSocketFactoryChannelCredentials.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.okhttp;
+
+import com.google.common.base.Preconditions;
+import io.grpc.ExperimentalApi;
+import javax.net.ssl.SSLSocketFactory;
+
+/** A credential with full control over the SSLSocketFactory. */
+@ExperimentalApi("There is no plan to make this API stable, given transport API instability")
+public final class SslSocketFactoryChannelCredentials {
+  private SslSocketFactoryChannelCredentials() {}
+
+  public static io.grpc.ChannelCredentials create(SSLSocketFactory factory) {
+    return new SslSocketFactoryChannelCredentials.ChannelCredentials(factory);
```

Comment:
The qualification `SslSocketFactoryChannelCredentials.` seems unnecessary. Or else we can name the inner class here something other than `ChannelCredentials` to avoid confusion. How about `LocalChannelCredentials`?

Reply:
Removed `SslSocketFactoryChannelCredentials.` prefix.